src/simplewallet/simplewallet.cpp

```diff
@@ -3030,12 +3069,25 @@ int main(int argc, char* argv[])
     }
 
     tools::wallet2 wal(testnet,restricted);
+    bool quit = false;
+    tools::signal_handler::install([&wal, &quit](int) {
+      quit = true;
+      wal.stop();
+    });
```

Comment:
That looks like a patch of mine that's already been merged. Merge error somewhere ?